modules/installation-requirements-user-infra-ibm-z-kvm.adoc

```diff
@@ -35,6 +35,11 @@ you install the cluster.
 To improve high availability of your cluster, distribute the control plane machines over different {op-system-base} instances on at least two physical machines.
 ====
 
+[IMPORTANT]
+====
+All virtual machines must reside in the same data store and in the same folder as the installer.
```

Comment:
s/data store/datastore
^This is a VMware/vSphere thing (even though I know ISG suggests two words).

Reply:
Avoid having stacked admonitions like this if possible. (It's not even ideal that there are already 2 stacked, but adding a third should be avoided.)

What about moving this addition to the end of the module?